packages/expo-cellular/README.md

```diff
@@ -21,6 +21,15 @@ For bare React Native projects, you must ensure that you have [installed and con
 expo install expo-cellular
 ```
 
+### Configure for Android
+
+This package requires the `android.permission.READ_PHONE_STATE` be added to your `AndroidManifest.xml`, this is used for `TelephonyManager` on Android. We **do not** require the more risky `READ_PRIVILEGED_PHONE_STATE` permission.
+
+```xml
+<!-- Added permissions -->
+<uses-permission android:name="android.permission.READ_PHONE_STATE" />
+```
```

Comment:
I think it would be helpful to note which methods require this permission

Reply:
won't do that in all of the other modules though.